.github/workflows/build.yml.in

```diff
@@ -147,6 +147,18 @@ jobs:
           lean --version
           lake --version
 
+      - name: build cache
+        run: |
+          lake build cache
+
+      - name: prune ProofWidgets .lake
+        run: |
+          # The ProofWidgets release contains not just the `.js` (which we need in order to build)
+          # but also `.oleans`, which may have been built with the wrong toolchain.
+          # This removes them.
+          rm -rf .lake/packages/proofwidgets/.lake/build/lib
+          rm -rf .lake/packages/proofwidgets/.lake/build/ir
```

Comment:
I think it might be worth cross-referencing a Zulip thread here, as this feels like a hack that is going to stop working in the not-too-distant future, and that ought to help with debugging it when that happens.

Otherwise this looks good to go.

Reply:
I've added a link. I'm hoping that @tydeu will later be able to come up with a better solution here, but that won't happen immediately.

Reply:
@semorrison At the moment, the ideal solution is just the Lake/Reservoir-cache which integrates all this. I am not sure if there is any better in-between solution.